entity-framework/core/managing-schemas/ensure-created.md

```diff
@@ -8,18 +8,18 @@ uid: core/managing-schemas/ensure-created
 ---
 # Create and Drop APIs
 
-The EnsureCreated and EnsureDeleted methods provide a lightweight alternative to [Migrations](xref:core/managing-schemas/migrations/index) for managing the database schema. These methods are useful in scenarios when the data is transient and can be dropped when the schema changes. For example during prototyping, in tests, or for local caches.
+The <xref:Microsoft.EntityFrameworkCore.Storage.IDatabaseCreator.EnsureCreated> and <xref:Microsoft.EntityFrameworkCore.Storage.IDatabaseCreator.EnsureDeleted> methods provide a lightweight alternative to [Migrations](xref:core/managing-schemas/migrations/index) for managing the database schema. These methods are useful in scenarios when the data is transient and can be dropped when the schema changes. For example during prototyping, in tests, or for local caches.
```

Comment:
Is it valid to do xrefs like this? They certainly don't look right when viewing on GitHub.,

Reply:
We do this in many places, e.g. https://github.com/dotnet/EntityFramework.Docs/blob/main/entity-framework/core/change-tracking/change-detection.md?plain=1#L129 (which looks fine [here](https://docs.microsoft.com/en-us/ef/core/change-tracking/change-detection#methods-that-automatically-detect-changes)). It's definitely not standard markdown (and so looks bad on github markdown rendering), but...

Am I missing something?